client/src/Home.js

```diff
@@ -0,0 +1,18 @@
+import {useState} from 'react';
+import ReviewList from './ReviewList';
+
+/**
+ * Homepage
+ * 
+ */
+
+function Home() {
+
+    return (
+        <div className = "home">
```

Comment:
Most of the time, people don't keep white spaces before and after `=`, so it is better to write it as:

```jsx
<div className="home">
```

Reply:
@auroravo2001 I still see same issue in `client/src/NavBar.js`, better to double check again.

Reply:
@jeromewu so sorry, it took me some time to figure out where it was...!

Reply:
No worries, I know it is hard to see them. Haha